### PR TITLE
[Feature] Truck illustration

### PR DESCRIPTION
**Pull request by jelledc**
### Added

- 240x120 truck

### CODE REVIEW

**Review by qubis741 — APPROVED** [review]

**Review by eniskraasniqi1 — APPROVED** [review]

**Review by lorgan3 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@teamleader/ui-illustrations",
   "description": "Teamleader UI Illustrations",
-  "version": "1.19.1",
+  "version": "1.20",
```
Comment (mrtus) [inline package.json:4]: Can you add the `v` to the version, since the tags are already like that too.

Comment (mrtus) [reply package.json:4]: Tags need to be created through CI now, which https://github.com/teamleadercrm/ui-visuals/pull/401 will facilitate

Comment (jelledc) [reply package.json:4]: Oh ok that explains why I'm having difficulties with the tags.

**Review by mrtus — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (jelledc) [reply package.json:4]: That is not a valid semver version though: https://docs.npmjs.com/cli/v10/configuring-npm/package-json

**Pull request merged**